modules/servicetemplate/src/main/java/org/eclipse/xpanse/modules/servicetemplate/ServiceTemplateManage.java

```diff
@@ -289,6 +294,43 @@ public String getOpenApiUrl(String id) {
         return openApiUrl;
     }
 
+    private void validateDeployVariable(List<DeployVariable> deployVariables) {
+        if (deployVariables.isEmpty()) {
```

Comment:
we do not need this validation. If something is not according to the data model specification, it will be automatically handled by Spring and rejected.